### ISSUE TITLE
Android Studio debug issue: Process finished with exit code 11

### ISSUE THREAD

**Issue by wuyuuu** [nihui/ncnn-android-nanodet#15]
Hi, 

First, thanks for your great work !

Recently I'm using your code to develop my project (for another deep learning-based vision task) and I want to modify the detect and post-process part at: https://github.com/nihui/ncnn-android-nanodet/blob/acb9dd3cbeb3b5e090c6baea73bba501b66cd2de/app/src/main/jni/nanodetncnn.cpp#L131-L133

Everything works fine for me when I go Run app. However, when I use debugger, the process would not stop at the abovementioned code, and I got Process finished with exit code 11.

Basically, I did not change any code so I'm wondering do you have similar issues in debugging? Or it could be some configuration problem refer to my Android Studio configuration.

Looking forward to your reply!

Thanks!

**Comment by wuyuuu** [comment]
Fixed by by substitute`ANativeWindow_getWidth(win);` with a constant. It seems to me this function does not work in debug mode.
